### ISSUE TITLE
test.py outputs Nan on MOT16 evaluation

### ISSUE THREAD

**Issue by Mushfequr-Rahman** [Zhongdao/Towards-Realtime-MOT#78]
After running test.py on MOT16 evaluation set with pretrained weights we get 0 and Nan on MOT metrics. 
         IDF1  IDP  IDR Rcll Prcn GT MT PT ML     FP FN IDs  FM  MOTA MOTP
MOT16-01 0.0% 0.0% nan% nan% 0.0%  0  0  0  0   4657  0   0   0 -inf%  nan
MOT16-03 0.0% 0.0% nan% nan% 0.0%  0  0  0  0  89459  0   0   0 -inf%  nan
MOT16-06 0.0% 0.0% nan% nan% 0.0%  0  0  0  0   9091  0   0   0 -inf%  nan
MOT16-07 0.0% 0.0% nan% nan% 0.0%  0  0  0  0  13491  0   0   0 -inf%  nan
MOT16-08 0.0% 0.0% nan% nan% 0.0%  0  0  0  0   9453  0   0   0 -inf%  nan
MOT16-12 0.0% 0.0% nan% nan% 0.0%  0  0  0  0   6697  0   0   0 -inf%  nan
MOT16-14 0.0% 0.0% nan% nan% 0.0%  0  0  0  0  11503  0   0   0 -inf%  nan
OVERALL  0.0% 0.0% nan% nan% 0.0%  0  0  0  0 144351  0   0   0 -inf%  nan
The video outputs appear to look normal.

**Comment by Zhongdao** [comment]
Are you running on MOT16 test set or training set? The test set does not include ground truth annotations so outputs should be like this.

**Comment by neversettle-tech** [comment]
> Are you running on MOT16 test set or training set? The test set does not include ground truth annotations so outputs should be like this.

So can you tell me where is the ground truth annotations of the MOT16 test set?

**Comment by liz2020** [comment]
> > Are you running on MOT16 test set or training set? The test set does not include ground truth annotations so outputs should be like this.
> 
> So can you tell me where is the ground truth annotations of the MOT16 test set?

There is no gt released for the test set. You can either evaluate on the train set, or submit your solution to the MOT16 challenge to get the result for the test set.
